frontend/packages/jobs/src/functions/processCreateKnowledgeSuggestion.ts

```diff
@@ -13,14 +13,17 @@ type CreateKnowledgeSuggestionPayload = {
   traceId?: string
 }
 
-export const processCreateKnowledgeSuggestion = async (
-  payload: CreateKnowledgeSuggestionPayload,
-) => {
-  const { projectId, type, title, path, content, branch, traceId } = payload
+type CreateKnowledgeSuggestionResult = {
+  suggestionId: number | null
+  success: boolean
```

Comment:
ask:`success: false` is possible in near future?


or  something like

```typescript
type CreateKnowledgeSuggestionResult = {
  suggestionId: number | null
  success: true
} | {
  suggestionId: null
  success: false
}
```

?

Reply:
That's right! More to the point, `suggestionId: number | null` looks better now.
I will refactor this!